Source/Core/Core/IOS/Network/Socket.h

```diff
@@ -184,6 +186,7 @@ class WiiSocket
   WiiSocket& operator=(WiiSocket&&) = default;
 
 private:
+  using Timeout = std::chrono::time_point<std::chrono::system_clock>;
```

Comment:
Sorry, I missed something important in my previous review: this should be using a steady_clock, not a system_clock. The latter isn't monotonic and can be changed at any time by the user.

Reply:
Good catch, I completely forgot about that.

I also updated the RFC comment since it's recommending this timeout to be **at least** 3 minutes.